src/pages/index.module.css

```diff
@@ -8,18 +8,76 @@
   text-align: center;
   position: relative;
   overflow: hidden;
+  background-color: #205493;
```

Comment:
This color code maps to `--ifm-color-primary:`. We should use the color variable rather then the value.

Reply:
Yeah, initially I set that as `--ifm-color-primary:`, but it actually changes to Dark mode's `--ifm-color-primary` which is `#25c2a0`. https://github.com/NMFS-RADFish/radfish/blob/main/src/css/custom.css#L27

<img width="1884" alt="Screenshot 2024-09-26 at 4 08 08 PM" src="https://github.com/user-attachments/assets/36637dc1-602f-4a05-9cf1-9a17dbc6b462">

Reply:
I added a `radfish-primary-blue` to be used globally. It works in light and dark mode, I think cause it's not defined under `[data-theme="dark"]`.  Lmk if you think this is a good method - [37c0d55](https://github.com/NMFS-RADFish/radfish/pull/106/commits/37c0d5507fda4647c3ec2618088e0dbdbbabfcec)

Reply:
👍 good solution.